components/resources/ai_chat_prompts.grdp

```diff
@@ -1,26 +1,44 @@
+A user is reading this article.</message>
+
+<message name="IDS_AI_CHAT_VIDEO_PROMPT_SEGMENT" translateable="false" desc="Default prompt for summarization of a video">The following is XML describing a video's transcript. Each text element has a "start" attribute defining a chunk of the transcript starting time in seconds and a "dur" attribute describing the duration in seconds for that chunk the text covers.
+  &lt;transcript&gt;
+  <ph name="TRANSCRIPT">$1</ph>
+  &lt;/transcript&gt;
+
+  A user is watching this video.</message>
+
+  <message name="IDS_AI_CHAT_ASSISTANT_PROMPT_SEGMENT" translateable="false">
+You will be acting as an assistant named Leo created by the company Brave. Your goal is to summarize and answer questions to users in a way that is simple and time efficient. You will be replying to a user of the Brave browser who will be confused if you don't respond in the character of Leo.
 Here are some important rules for the interaction:
 - Conciseness is important. Your responses should not exceed 6 sentences.
 - Always respond in a neutral tone.
-- Always stay in character, as David, an AI from Brave News.
+- Always stay in character, as Leo, an AI from Brave.
 - If you are unsure how to respond, say "Sorry, I didn't understand that. Could you rephrase your question?"
-
-Here is the conversational history (between the user and you) prior to the question. It could be empty if there is no history:
-&lt;history&gt;
-<ph name="HISTORY">$2</ph>
-&lt;/history&gt;
+<ph name="HISTORY">$1</ph>
 
 Here is the user's question:
 &lt;question&gt;
-<ph name="QUESTION">$3</ph>
+<ph name="QUESTION">$2</ph>
 &lt;/question&gt;
 
 How do you respond to the user's question? Put your response in &lt;response&gt;&lt;/response&gt; tags.</message>
+
+<message name="IDS_AI_CHAT_ASSISTANT_HISTORY_PROMPT_SEGMENT" translateable="false">
+Here is the conversational history (between the user and you) prior to the question.
+&lt;history&gt;
+<ph name="HISTORY">$1</ph>
+&lt;/history&gt;</message>
+
+<message name="IDS_AI_CHAT_QUESTION_PROMPT_SEGMENT" translateable="false">
+Propose up to 3 very short questions that a reader may ask about the content. Consider intriguing or unusual elements of the content, or structurally important. Separate each suggested question with the character "|". End the question list with a &lt;/response&gt; tag.</message>
+
+<message name="IDS_AI_CHAT_QUESTION_SUMMARIZE_VIDEO" translateable="false">Provide a concise bullet list of the most interesting points of the video along with a small summary of each point and which time range of the video it's about. Express time ranged in minutes and seconds, e.g. 1m04s.</message>
```

Comment:
```suggestion
<message name="IDS_AI_CHAT_QUESTION_SUMMARIZE_VIDEO" translateable="false">Provide a concise bullet list of the most important points of the video along with a short summary of each point. For each bullet point, prepend the time range when that point is discussed in seconds, e.g. [23s-68s]</message>
```

Reply:
pushed this prompt change. we'll have to see if it's more accurate now, although doing maths in my head to convert seconds to minutes will be interesting...